docs/data/base/components/snackbar/snackbar.md

```diff
@@ -0,0 +1,34 @@
+---
+product: base
+title: Unstyled React Snackbar component
+components: SnackbarUnstyled
+githubLabel: 'component: snackbar'
+packageName: '@mui/base'
+---
+
+# Unstyled snackbar
```

Comment:
@samuelsycamore can help with the docs around the base components :)

Reply:
I'm happy to handle copy editing on this once we're satisfied with the formatting and structure of these pages in #33156. I should have new drafts of those pages ready by the end of the day on my side of the planet :)